eng/mgmt/automation/parameters.py

```diff
@@ -69,12 +69,58 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.azure</groupId>
-  <artifactId>azure-{0}-service</artifactId>
+  <artifactId>azure-{service}-service</artifactId>
   <packaging>pom</packaging>
   <version>1.0.0</version><!-- Need not change for every release-->
-  <modules>
-  </modules>
+
+  <profiles>
+    <profile>
+      <id>coverage</id>
+      <modules>
+        <module>{artifact_id}</module>
+      </modules>
+
+      <dependencies>
+        <dependency>
+          <groupId>{group_id}</groupId>
+          <artifactId>{artifact_id}</artifactId>
+          <version>1.0.0-beta.1</version> <!-- {{x-version-update;{group_id}:{artifact_id};current}} -->
```

Comment:
Should this version be a variable?

Reply:
Yes, but actually it will be updated by `update_version` step, so I left it constant.